docs/script.js

```diff
@@ -58,51 +85,123 @@ $.getJSON('files.json')
         if (attr in format.files)
           options(q, attr, node[attr])
       })
-      options(q, 'ext', ['svg', 'png'])
-      options(q, 'mirror', { '': '', 'mirror': '1' })
+
       $('.comicgen-attrs .wip').remove()
-      comicgen('.target', q)
-      $('.target-container').css({ width: q.width + 'px', height: q.height + 'px' })
+
+      $('.download-png').off('click').on('click', function download() {
+        var image = new Image()
+        image.src = canvas.toDataURL('png')
+        window.open('').document.write(image.outerHTML)
+      })
+
+      $('.download-svg').off('click').on('click', function download() {
+        window.open('').document.write(canvas.toSVG())
+      })
+
+      $('body')
+        .off('keyup')
+        .on('keyup', function (e) { if (e.keyCode == 46) object_action(canvas, 'delete') })
+
+
+      var _attrs = Object.assign({}, comicgen.defaults, q)
+      
+      deleteCharacter(canvas, q.name)
```

Comment:
FYI: Soon, we will want to have multiple instances of the same character in the same panel.